scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -111,12 +114,14 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         nodememory = args.controllernodememory
 
     raidvolume = ""
+    serialre = re.compile("[^{0}]".format(QEMU_SERIAL_PARAM_ACCEPTED_CHARS))
```

Comment:
Why not just hardcode it as
```python
serialre = re.compile("[^A-Za-z0-9-_]")
```
which improves readability by reduced indirection